src/classes/FormsApps.ts

```diff
@@ -45,6 +45,11 @@ export default class FormsApps extends OneBlinkAPI {
    * ```
    *
    * @param token The JWT you wish to verify
+   * @returns
+   *
+   *   ## Role Permissions Required
+   *
+   *   No Permissions Required
```

Comment:
I think remove this as these are static functions, there is not key being used. It is clear there are not permissions required.